infra/modules/providers/azure/app-gateway/README.md

```diff
@@ -0,0 +1,92 @@
+    - tier : The Tier of the SKU to use for this Application Gateway. Possible values are Standard, Standard_v2, WAF and WAF_v2.
+    - capacity : The Capacity of the SKU to use for this Application Gateway - which must be between 1 and 10.
+  - gateway_ip_configuration
+    - name : The Name of this Gateway IP Configuration.
+    - subnet_id : The ID of a Subnet.
+  - frontend_port
+    - name : The name of the Frontend Port.
+    - port : The port used for this Frontend Port.
+  - frontend_ip_configuration
+    - name : The name of the Frontend IP Configuration.
+    - public_ip_address_id : The ID of a Public IP Address which the Application Gateway should use.
+  - backend_address_pool
+    - name : The name of the Backend Address Pool.
+  - backend_http_settings
+    - name : The name of the Backend HTTP Settings Collection.
+    - cookie_based_affinity : Is Cookie-Based Affinity enabled? Possible values are Enabled and Disabled.
+    - port : The port which should be used for this Backend HTTP Settings Collection.
+    - protocol : The Protocol which should be used. Possible values are Http and Https.
+  - http_listener
+    - name : The Name of the HTTP Listener.
+    - frontend_ip_configuration_name : The Name of the Frontend IP Configuration used for this HTTP Listener.
+    - frontend_port_name : The Name of the Frontend Port use for this HTTP Listener.
+    - protocol : The Protocol to use for this HTTP Listener. Possible values are Http and Https.
+  - request_routing_rule
+    - name : The Name of this Request Routing Rule.
+    - rule_type : The Type of Routing that should be used for this Rule. Possible values are Basic and PathBasedRouting.
+    - http_listener_name : The Name of the HTTP Listener which should be used for this Routing Rule.
+    - backend_address_pool_name : The Name of the Backend Address Pool which should be used for this Routing Rule. Cannot be set if redirect_configuration_name is set.
+    - backend_http_settings_name : The Name of the Backend HTTP Settings Collection which should be used for this Routing Rule. Cannot be set if redirect_configuration_name is set.
+
+
+Please click the [link](https://www.terraform.io/docs/providers/azurerm/r/application_gateway.html) to get additional details on settings in Terraform for Azure Application Gateway.
+
+## Usage
+
+### Module Definitions
+
+- Service Plan Module        : infra/modules/providers/azure/service-plan
+- Virtual Network Module     : https://github.com/Microsoft/bedrock/tree/master/cluster/azure/vnet
+- Application Gateway Module : infra/modules/providers/azure/app-gateway
+
+```
+module "service_plan" {
+  resource_group_name     = "test-rg"
+  resource_group_location = "eastus"
+  service_plan_name       = "test-svcplan"
+}
+
+module "vnet" {
+  vnet_name               = "test-vnet"
```

Comment:
Aren’t we missing the module source attribute?

Reply:
Same response as PR #80